docs/dev/system_keyspace.md

```diff
@@ -215,6 +217,9 @@ Only tables which use tablet-based replication strategy have an entry here.
 `tablet_count` is the number of tablets in the map.
 `table_name` is the name of the table, provided for convenience.
 
+`resize` is the resize decision type that spans all tablets of a given table, which can be one of: `merge`, `split` or `none`.
+`resize_phase` is the timestamp of the resize decision, and it helps to distinguish older (stale) resize decisions from newer ones.
```

Comment:
please rename it. A phase isn't a timestamp.

Reply:
I discussed it with @tgrabiec. We thought of renaming it to generation.